bundles/org.openhab.binding.feed/README.md

```diff
@@ -29,13 +29,17 @@ Required configuration:
 Optional configuration:
 
 - **refresh** - a refresh interval defines after how many minutes the binding will check, if new content is available. Default value is 20 minutes
+- **number-of-entries** - the number of entries to create channels for
```

Comment:
```suggestion
- **numberOfEntries** - the number of entries to create channels for
```

or `numberofentries`. I think all bindings use either all-lower-case or camel-case for parameters.